LKMEX-whitelist/ElrondBuddies.md

```diff
@@ -0,0 +1,48 @@
+## Name of the project
+Elrond Buddies
+
+## Description of the project
+Elrond Buddies is a NFT project composed of 10,000 unique crypto collectibles on MultiversX.
+
+website: https://www.elrondbuddies.com/
+
+**Old PR**: https://github.com/ElrondNetwork/elrond-specs/pull/19
+
+## Purpose
+We send to our holders monthly rewards with LKMEX, UTK, RIDE, ITHEUM and WATER.
+
+To farm LKMEX on the dex we use this following wallet called 'ebuddiesdao', every month 20% of what we generated is reinvested in the LKMEX farm and we distribute 80% to the holders.
+https://explorer.elrond.com/accounts/erd16n5yll0yver02j4wm0l5fzw79c0dqg8ra3exq3ag35n7tfuwwxksxxzwzt
+
+We started distributing LKMEX on the 21/04/2022, so far we have distributed 73,002,098 LKMEX to our holders and reinvested 14,756,856 LKMEX
+
+We used two python scripts that we developed to get the holders and to distribute them the LKMEX:
+
+https://github.com/xdevguild/multiversX-nft-holders
+https://github.com/xdevguild/esdt-and-lkmek-airdrop-scripts
+
+Now that you guys want us to use a SC to distribute LKMEX we found a new way to do it with it.
+
+## Address of Smart Contract (Devnet)
+https://devnet-explorer.elrond.com/accounts/erd1qqqqqqqqqqqqqpgqwk5lgpf2fdf2veej683ltr2gzcyr0al5y7zswazjc5
+
+When you reviewed this SC we will deploy it on Mainnet. 
+
+## Smart Contract Code
+https://github.com/xdevguild/sc-multi-sender-rs
```

Comment:
Multi-send smart contract is not accepted. You need to create contracts where it is clear that it is used for NFT staking. The contract is too general and can be used for wrong secondary markets.

Reply:
@sasurobert, WHAT ? 

This is what you told us last time :

> We do not whitelist user addresses, only smart contracts. Distribution of tokens can happen through SCs. You can collaborate with other projects who have smart contracts and help you distribute through those.

You never talked about 'staking'  ! You said "We do not whitelist user addresses, only smart contracts. Distribution of tokens can happen through SCs" So we have found a Smart Contracts to make the distribution happen through a Smart Contracts, how come your are changing the condition now ? I have waited for 2 weeks, loosed 2 months working on this, and you change the condition at the last time !!!!!! We are the ones entertaining you tool and this how you respect us ?! I am really disappointed !!

I don't want to implement staking for my collections it goes against our vision ! 

I know you will never read this message because you just don't care about the people spending energies on your plateforme, I am really disappointed ! 

<img width="868" alt="Capture d’écran 2023-01-10 à 09 30 31" src="https://user-images.githubusercontent.com/23435882/211500244-b23bc047-ea4e-45bc-bfbd-4a62394d1553.png">

Reply:
@sasurobert expecting an answer ⬆

Reply:
I feel it, you will cowardly abandon us and close this PR

Reply:
😡 you guys ghosting us like we mean nothing to you, keep on disrespecting people you might learn something one day. I never wanted to be disappointed by MultiversX team, but now it's too late, thank you @sasurobert